### ISSUE TITLE
Fedora 28 - "conn.go:173:12: error: incompatible types in assignment"

### ISSUE THREAD

**Issue by przerada** [emersion/neutron#54]
Hi guys, at first command get a problem, this is what I've got

`$

 go get -u github.com/emersion/neutron
# github.com/emersion/neutron/backend/imap
go/src/github.com/emersion/neutron/backend/imap/conn.go:173:12: error: incompatible types in assignment
  c.Updates = updates
            ^
`
I was looking for similar issues but with no luck.
Greetings

**Comment by emersion** [comment]
Yeah, this repo needs to be updated. Unfortunately I only work on open-source projects in my spare time so I won't be able to do this anytime soon. Pull requests are welcome though.

**Comment by bitterdev** [comment]
On which ubuntu version is this working?
